build_sys_common.rs

```diff
@@ -0,0 +1,31 @@
+use std::env::var;
+
+// Attempts to discover which library file to use and where it is located.
+//
+// If the `cfg(target_env)` is set to "MSVC" then it will use the MSVC
+// libraries, otherwise it will assume "GNU" and use the GCC libraries.
+//
+// If the environment variable `CSFML_HOME` is set, then it will look there
+// for the libraries, otherwise it defaults to `.`.
+//
+// It assumes that CSFML is simply unzipped - as is - into a folder, and the
+// libraries exist in: `$CSFML_HOME/lib/[msvc|gcc]/*.lib`.
+//
+pub fn lib_path(file: &str) -> String {
+    let lib = if cfg!(target_env = "msvc") {
+        format!("csfml-{}", file)
+    } else {
+        format!("libcsfml-{}", file)
```

Comment:
I don't have Linux to test under. So, it might need another cfg! check, sadly. But, if just download CSFML from the main page and extract the files, in the lib/gcc folder is "libcsfml-audio.a", "libcsfml-graphics.a", etc.